### PR TITLE
Refactor Marathon task details into own component

### PR DESCRIPTION
**Pull request by aldipower**
This is a preliminary PR to be able to extend the Marathon task details in a meaningful and clean way.

Please merge https://github.com/dcos/dcos-ui/pull/263 before.

### CODE REVIEW

**Comment by rcorral** [comment]
retest this please

**Comment by aldipower** [comment]
done

File: src/js/components/MarathonTaskDetailsList.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import DescriptionList from './DescriptionList';
+import MarathonStore from '../stores/MarathonStore';
+
+class MarathonTaskDetailsList extends React.Component {
+  constructor() {
+    super();
+  }
```
Comment (philipnrmn) [inline src/js/components/MarathonTaskDetailsList.js]: I think this constructor can be removed completely.

**Comment by philipnrmn** [comment]
[One pick](https://github.com/dcos/dcos-ui/pull/262#discussion_r65186776), otherwise I'm happy to merge.

**Pull request merged**